homeworks/01-bubble-sort/src/App.tsx

```diff
@@ -0,0 +1,109 @@
+import React, {Component} from 'react'
+import Button from "./components/button/Button"
+import Column from "./components/column/Column"
+import './App.css'
+
+type AppState = {
+	statusOfGame: string,
+	statusOfBtn: string,
+	arrOfNumbers: number[],
+	arrOfTimerID: number[],
+	disabled: boolean,
+}
+
+class App extends Component<{}, AppState> {
+
+  private timerID: any
+
+	state = {
+		statusOfGame: 'not solved',
```

Comment:
You have different indentations in your code (2, 4, and 8 spaces at the same time). Looks strange, try to configure `.editorconfig` or/and `eslint`.

Reply:
fixed